src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -85,7 +98,7 @@ public PatchXpathFilterElement(String fileName,
         this.lineRangeList = lineRangeList;
         this.strategy = strategy;
         if (checkNameForContextStrategyByTokenOrParent != null) {
-            this.checkNamesForContextStrategyByTokenOrParentSet.addAll(
+            this.checkNamesForContextStrategyByTokenOrAncestorSet.addAll(
```

Comment:
you should rename `checkNameForContextStrategyByTokenOrParent` too

Reply:
done